README.md

```diff
@@ -78,9 +78,9 @@ repos on GitHub, it is still a useful source of relative usage data.
 
 ## Contributing
 
-Take a look at [some of these issues](https://github.com/divad12/vim-awesome/issues?labels=easyfix&state=open) to get started.
+Take a look at [some of these issues](https://github.com/vim-awesome/vim-awesome/issues?labels=easyfix&state=open) to get started.
 
-Chat with us on [Gitter](https://gitter.im/divad12/vim-awesome)!
+Chat with us on [Gitter](https://gitter.im/vim-awesome/vim-awesome)!
```

Comment:
I'll do a little test. I'll create a throw-away repo, get it running on Gitter, and then attempt to move it to an Org and see what happens.